### PR TITLE
[prod nightly] 7.13.0

### PR DESCRIPTION
**Pull request by Ginxo**
Product nightly for main branches is now 7.13.0

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

**Comment by mareknovotny** [comment]
looks good so that https://github.com/kiegroup/kie-jenkins-scripts/pull/1118/files#diff-9d55ce9075a526a3684617ae54e1f15ee81673ba5c89794b916086a8ffacc4d9R40 can be updated always on the top of job, which creates the new branches

**Comment by Ginxo** [comment]
> looks good so that https://github.com/kiegroup/kie-jenkins-scripts/pull/1118/files#diff-9d55ce9075a526a3684617ae54e1f15ee81673ba5c89794b916086a8ffacc4d9R40 can be updated always on the top of job, which creates the new branches
https://issues.redhat.com/browse/BXMSPROD-1483

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 1]
```diff
@@ -25,7 +26,7 @@ pipeline{
                 build job: 'nightly/main', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'KIE_GROUP_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-main/content-compressed'],
                         [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: 'main'],
-                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.12.0'],
+                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: "${PROD_VERSION}"],
```
Comment (mbiarnes) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:29]: @Ginxo shouldn't this be ${PRODUCT_VERSION} ??

**Review by mbiarnes — COMMENTED** [review]
@Ginxo shouldn't this be ${PRODUCT_VERSION}

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy (hunk 1 above)
Comment (Ginxo) [reply job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:29]: @mbiarnes you are right

**Review by mareknovotny — APPROVED** [review]

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 2]
```diff
@@ -94,7 +95,7 @@ pipeline{
                 build job: 'kogito-tooling.nightly/0.13.0-prerelease', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-7.12/content-compressed'],
                         [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '712'],
-                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.12.0'],
+                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: "${PROD_VERSION}"],
```
Comment (mareknovotny) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:98]:
```suggestion
                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: "${PROD_VERSION}"],
```

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 3]
```diff
@@ -65,7 +66,7 @@ pipeline{
                 build job: 'nightly/7.59.x', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'KIE_GROUP_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-7.12/content-compressed'],
                         [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '712'],
-                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.12.0'],
+                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: "${PRODUCT_VERSION}"],
```
Comment (winklerm) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:69]: Is this correct? UMB_VERSION should match as well.

Comment (winklerm) [reply job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:69]: I see, UMB_VERSION is correct, but PRODUCT_VERSION should not be used in this case.

Comment (mareknovotny) [reply job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:69]: good catch

**Pull request merged**